generic/tclcmark.c

```diff
@@ -244,6 +246,12 @@ static int tclcmark_render_cmd(
             goto vamoose;
     }
 
+    if (gfm || tasklist) {
+        fprintf(stderr, "Using tasklist extensions");
```

Comment:
Why is this fprintf here? Left over?

Reply:
yep sorry. I wasn't sure why "- [ ] abc" didn't work, which was silly driver error :).